packages/main/src/themes/base/Input-parameters.css

```diff
@@ -61,4 +63,17 @@
 	--_ui5_input_value_state_error_warning_placeholder_font_weight: normal;
 	--_ui5_input_focus_outline_color: var(--sapContent_FocusColor);
 	--_ui5-input_error_placeholder_color: var(--sapField_PlaceholderTextColor);
+	--_ui5-input-icons-count: 0;
+	--_ui5_input_icon_width: 2.25rem;
+	--_ui5_input_has_tokens: 0;
+	--_ui5_input_token_min_width: 4.125rem;
+	--_ui5_input_tokenizer-width: 0;
+}
+
+[data-ui5-compact-size],
+.ui5-content-density-compact,
+.sapUiSizeCompact {
+	--_ui5_input_min_width: 2rem;
+	--_ui5_input_icon_width: 2rem;
+	--_ui5_input_token_min_width: 3.5625rem;
```

Comment:
Should we include --_ui5_input_margin_top_bottom here aswell and remove it from the parameters in each theme ?